src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java

```diff
@@ -574,6 +578,15 @@ private String getMessageName(Schema sch) {
     return messageName;
   }
 
+  private String getMessageName(Schema sch, Boolean isEnum) {
+    String messageName = sch.getIdentifier();
+    // For the enum name start with uppercase letter, add the underscore '_' suffix.
+    if (isEnum && Character.isUpperCase(messageName.charAt(0))) {
```

Comment:
I think just starting with capital letter is the criteria.

`IpStuff` as `enum`
`Ip_stuff` as field name

this will one produce IpStuff and IpStuff in node, won't it?

Reply:
> 
`IpStuff` as `enum`
`Ip_stuff` as `field name`

Field Node is still become `IpStuff`, and it cause same problem, duplicate with enum name `IpStuff`.

Reply:
If we have discovery json file `IpStuff` property for enum, it will change enum name to `IpStuffEnum`, and field is `Ip_stuff`. `probufjs` is ok with it.

Reply:
Clarify above case, as long as discovery JSON's the property name of enum start with uppercase letter, the proto enum name will add `Enum` as suffix. The reference field name keep the same.

This resolved the conflict in `protobufjs` generated in `protos.json`